packages/components/src/Badge/_docs/Badge.stickersheet.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Badge } from "../index"
+
+export default {
+  title: "KAIO/Badge",
```

Comment:
Wait, why are these all under `KAIO` and not `Components`? 🤔

Reply:
Changing this soon but the general idea is to put them all into a path we can hide for now:

https://github.com/cultureamp/kaizen-design-system/pull/4043